2017-07-24-tidbbestpractice.md

```diff
@@ -173,7 +173,7 @@ sync-log = true
 As mentioned before, TiDB limits the size of a single transaction in the Key-Value layer. As for the SQL layer, a row of data is mapped to a Key-Value entry. For each additional index, there will be one more Key-Value entries. So the limits mirrored in the SQL layer are:
```

Comment:
So the limits mirrored in the SQL layer are: -> So the limits mirrored in the SQL layer for a single transaction are: